test/plugin/test_out_elasticsearch_data_stream.rb

```diff
@@ -437,6 +437,38 @@ def test_datastream_configure
     assert_equal "foo", driver(conf).instance.data_stream_name
   end
 
+  def test_hosts_list_configure
+    config = %{
+      hosts            https://john:password@host1:443/elastic/,http://host2
+      path             /default_path
+      user             default_user
+      password         default_password
+      data_stream_name default
+    }
+    stub_elastic_info("https://host1:443/elastic//", "7.9.0",
+                         {'Authorization'=>'Basic am9objpwYXNzd29yZA=='})
```

Comment:
`"Authorization" => "Basic #{Base64.encode64('john:password').split.first}"` or something may be readable.

Reply:
Fixed in https://github.com/uken/fluent-plugin-elasticsearch/pull/942/commits/67390bcfa1ece1d42a56d3584ce14d18b902c561